NoClassDefFoundError: net/milkbowl/vault/economy/Economy

**I don't need economy on the server, but even when I disable this feature in the config file, the error remains.**

`[20:51:37] [Server thread/ERROR]: Could not pass event ServiceRegisterEvent to EzAfk v2.4.0
java.lang.NoClassDefFoundError: net/milkbowl/vault/economy/Economy
        at ezafk-2.4.0.jar//com.gyvex.ezafk.listener.EconomyServiceListener.isEconomyService(EconomyServiceListener.java:53)
        at ezafk-2.4.0.jar//com.gyvex.ezafk.listener.EconomyServiceListener.onServiceRegister(EconomyServiceListener.java:19)
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71)
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54)
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131)
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628)
        at org.bukkit.plugin.SimpleServicesManager.register(SimpleServicesManager.java:59)
        at Chunky-Bukkit-1.4.40.jar//org.popcraft.chunky.ChunkyBukkit.onEnable(ChunkyBukkit.java:62)
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:279)
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:207)
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109)
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520)
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:639)
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:596)
        at net.minecraft.server.MinecraftServer.initPostWorld(MinecraftServer.java:636)
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:365)
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1253)
        at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:388)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.ClassNotFoundException: net.milkbowl.vault.economy.Economy
        ... 20 more`

--- a/src/main/java/com/gyvex/ezafk/bootstrap/Bootstrap.java
+++ b/src/main/java/com/gyvex/ezafk/bootstrap/Bootstrap.java
@@ -84,11 +84,13 @@ public class Bootstrap {
         // Storage repository is initialized during Registry.init(); on reload the command will refresh it.
         AfkTimeManager.load(plugin);
 
-        economyServiceListener = new EconomyServiceListener();
-        try {
-            plugin.getServer().getPluginManager().registerEvents(economyServiceListener, plugin);
-        } catch (org.bukkit.plugin.IllegalPluginAccessException ex) {
-            plugin.getLogger().warning("Failed to register EconomyServiceListener during enable: " + ex.getMessage());
+        if (plugin.getConfig().getBoolean("economy.enabled", false)) {
+            economyServiceListener = new EconomyServiceListener();
+            try {
+                plugin.getServer().getPluginManager().registerEvents(economyServiceListener, plugin);
+            } catch (org.bukkit.plugin.IllegalPluginAccessException ex) {
+                plugin.getLogger().warning("Failed to register EconomyServiceListener during enable: " + ex.getMessage());
+            }
         }
 
         registerListener(new MoveListener(plugin));

--- a/src/main/java/com/gyvex/ezafk/listener/EconomyServiceListener.java
+++ b/src/main/java/com/gyvex/ezafk/listener/EconomyServiceListener.java
@@ -50,7 +50,11 @@ public class EconomyServiceListener implements Listener {
     }
 
     private boolean isEconomyService(RegisteredServiceProvider<?> provider) {
-        return provider != null && Economy.class.equals(provider.getService());
+        try {
+            return provider != null && Economy.class.equals(provider.getService());
+        } catch (NoClassDefFoundError e) {
+            return false;
+        }
     }
 
     private com.gyvex.ezafk.integration.EconomyIntegration getEconomyIntegration() {

